src/main/java/at/hannibal2/skyhanni/features/combat/damageindicator/MobFinder.kt

```diff
@@ -71,8 +73,82 @@ class MobFinder {
     // F5
     private var floor5lividEntity: EntityOtherPlayerMP? = null
     private var floor5lividEntitySpawnTime = 0L
-    private val correctLividPattern =
-        "§c\\[BOSS] (.*) Livid§r§f: Impossible! How did you figure out which one I was\\?!".toPattern()
+
+    private val patternGroup = RepoPattern.group("mob.finder")
+    private val correctLividPattern by patternGroup.pattern(
+        "lividpattern",
+        "§c\\[BOSS] (.*) Livid§r§f: Impossible! How did you figure out which one I was\\?!"
```

Comment:
not something you added but this should either have a named capture group or none at all